src/database-layer/db.ts

```diff
@@ -218,20 +226,18 @@ class AutomaticClose {
 	constructor(
 		tx: Tx,
 		private stackTraceErr?: Error,
+		timeoutMS = env.db.timeoutMS,
 	) {
 		this.automaticClose = () => {
 			console.error(
-				`Transaction still open after ${env.db.timeoutMS}ms without an execute call.`,
+				`Transaction still open after ${timeoutMS}ms without an execute call.`,
 			);
 			if (this.stackTraceErr) {
 				console.error(this.stackTraceErr.stack);
 			}
 			void tx.rollback();
 		};
-		this.automaticCloseTimeout = setTimeout(
-			this.automaticClose,
-			env.db.timeoutMS,
-		);
+		this.automaticCloseTimeout = setTimeout(this.automaticClose, timeoutMS);
```

Comment:
I can't comment on the relevant line but there's other uses of `env.db.timeoutMS` that also need updating